pyodbc.pyi,sha256=JWNpdgmrxhNioTjW6OoAEcXELcGZobcMYzKtYlcJRuY,32320
pyodbc.cpython-36m-x86_64-linux-gnu.so,sha256=HrBbyLOIMZ8MvMaBOIVUA1kmAlnGQb-u1St54n37eNs,907568
pyodbc-4.0.38.dist-info/WHEEL,sha256=YVAXCacQXnznQ-_lMj8Nvqrs7YPtMP1pHRCw9EcyWRQ,150
pyodbc-4.0.38.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-4.0.38.dist-info/RECORD,,
pyodbc-4.0.38.dist-info/METADATA,sha256=uQiX0gZrhbZAadzgdRRPh43nQLecgG0BXB_DuOHqiiY,3180
pyodbc-4.0.38.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
